benchmarks/bench_sdbaseline.py

```diff
@@ -671,60 +600,45 @@ def _refetch_files(self, files, from_dir=None):
         self._remove(files)
         self._copy(files, from_dir)
     
-    def time_testVariable00(self):
-        """Test blfunc='variable' with variable baseline functions and orders"""
+    def time_variable_orders(self):
+        """sdbaseline: Test blfunc='variable' with variable baseline functions and orders - testVariable00"""
         print('why is this failing?')
         infile = 'analytic_variable.ms'
         self.paramfile = 'analytic_variable_blparam.txt'
         self._refetch_files([infile, self.paramfile], self.datapath)
         sdbaseline(infile=infile, blfunc='variable', outfile=self.outfile, blparam=self.paramfile, datacolumn=self.column)
 
 
-class sdbaseline_updateweightTest(sdbaseline_unittest_base):
+class UpdateWeight(BaseSetup):
     """
     Tests for updateweight=True
     to confirm if WEIGHT_SPECTRUM column is removed
-    """
-
-    datapath = ctsys_resolve('unittest/sdbaseline/')
-    infile = 'uid___A002_X6218fb_X264.ms'
-    outroot = sdbaseline_unittest_base.taskname+'_updateweighttest'
-    outfile = outroot + '.ms'
-
-    def setUp(self):
-        remove_files_dirs(self.infile)
-        shutil.copytree(os.path.join(self.datapath, self.infile), self.infile)
-
-    def tearDown(self):
-        remove_files_dirs(self.infile)
-        remove_files_dirs(self.outroot)
-
-    def time_test000(self):
-        sdbaseline(infile=self.infile, outfile=self.outfile, intent='OBSERVE_TARGET#ON_SOURCE', spw='9', datacolumn='data', updateweight=True)
-
-
-class sdbaseline_updateweightTest2(sdbaseline_unittest_base):
-    """
-    Tests for updateweight=True cases
     test052 --- blmode='apply', spw to flag channels 4500-6499
     """
 
     datapath = ctsys_resolve('unittest/sdbaseline/')
-    infile = 'analytic_order3_withoffset.ms'
-    outroot = sdbaseline_unittest_base.taskname + '_updateweighttest'
+    infile = 'uid___A002_X6218fb_X264.ms'
+    infile2 = 'analytic_order3_withoffset.ms'
+    outroot = BaseSetup.taskname+'_updateweighttest'
     outfile = outroot + '.ms'
     spw = '*:0~4499;6500~8191'
 
     def setUp(self):
         remove_files_dirs(self.infile)
         shutil.copytree(os.path.join(self.datapath, self.infile), self.infile)
+        remove_files_dirs(self.infile2)
+        shutil.copytree(os.path.join(self.datapath, self.infile2), self.infile2)
 
     def tearDown(self):
         remove_files_dirs(self.infile)
+        remove_files_dirs(self.infile2)
         remove_files_dirs(self.outroot)
 
-    def time_test052(self):
-        sdbaseline(infile=self.infile, outfile=self.outfile, intent='OBSERVE_TARGET#ON_SOURCE', datacolumn='float_data', spw=self.spw)
+    def time_remove_weight_col(self):
+        sdbaseline(infile=self.infile, outfile=self.outfile, intent='OBSERVE_TARGET#ON_SOURCE', spw='9', datacolumn='data', updateweight=True)
+    
+    def time_blmode_apply_spw_to_flag(self):
```

Comment:
Can you please add a doc string for this test? The one from the original test is fine.
'''sdbaseline: blmode='apply', spw to flag channels 4500-6499 - test052'''